### PR TITLE
Documentation for Logitech UE Smart Radios

### PR DESCRIPTION
**Pull request by GreenTurtwig**
**Description:** Documentation for Logitech UE Smart Radios.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** 
https://github.com/home-assistant/home-assistant/pull/10077

:cat2:

### CODE REVIEW

File: source/_components/media_player.ue_smart_radio.markdown [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: ue_smart_radio
+    email: EMAIL
+    password: PASSWORD
+```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/media_player.ue_smart_radio.markdown]: Please use the configuration tags as described on our guidelines:

https://home-assistant.io/developers/documentation/create_page/#configuration

Comment (GreenTurtwig) [reply source/_components/media_player.ue_smart_radio.markdown]: Done, good that it's all standardised now. :+1:

**Comment by frenck** [comment]
@GreenTurtwig Thank you for updating! Approved 🏅

File: source/_components/media_player.ue_smart_radio.markdown [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+title: "Logitech UE Smart Radio"
+description: "Instructions on how to integrate a Logitech UE Smart Radio player into Home Assistant."
+date: 2017-11-20 11:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ueradio.png
+ha_category: Media Player
+ha_release: 0.59
```
Comment (MartinHjelmare) [inline source/_components/media_player.ue_smart_radio.markdown]: `"0.60"` nb keep the qoutes.

File: source/_components/media_player.ue_smart_radio.markdown [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+The `ue_radio` platform allows you to control a [Logitech UE Smart Radio](https://www.uesmartradio.com) from Home Assistant. This lets you control both Logitech UE Smart Radios and Logitech Squeezebox Radios that have been updated with the UE Smart Radio update.
+
+To add your UE Smart Radio player to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: ue_smart_radio
+    email: EMAIL
```
Comment (MartinHjelmare) [inline source/_components/media_player.ue_smart_radio.markdown]: Update to username.

**Comment by GreenTurtwig** [comment]
Totally forgot about updating the docs, done now though. :+1:

**Pull request merged**